<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BigViewWindow</class>
 <widget class="QMainWindow" name="BigViewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>828</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Multiple Image Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGraphicsVolumeView" name="graphicsView">
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="backgroundBrush">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
       <property name="dragMode">
        <enum>QGraphicsView::ScrollHandDrag</enum>
       </property>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Intensity</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDualSlider" name="intensitySlider">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QComboBox" name="fileList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>139</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad"/>
   <addaction name="separator"/>
   <addaction name="actionIJ"/>
   <addaction name="actionJK"/>
   <addaction name="actionKI"/>
   <addaction name="separator"/>
   <addaction name="actionLR"/>
   <addaction name="actionUD"/>
   <addaction name="separator"/>
   <addaction name="actionDualView"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionShowMenu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="bigViewWindow.qrc">
     <normaloff>:/Icons/Images/Photos-icon.png</normaloff>:/Icons/Images/Photos-icon.png</iconset>
   </property>
   <property name="text">
    <string>Show Menu</string>
   </property>
   <property name="toolTip">
    <string>Show Intensity Control Menu</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="bigViewWindow.qrc">
     <normaloff>:/Icons/Images/folder-icon.png</normaloff>:/Icons/Images/folder-icon.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load an image</string>
   </property>
  </action>
  <action name="actionIJ">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>IJ</string>
   </property>
   <property name="toolTip">
    <string>Axial View</string>
   </property>
  </action>
  <action name="actionJK">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>JK</string>
   </property>
   <property name="toolTip">
    <string>Sagittal View</string>
   </property>
  </action>
  <action name="actionKI">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>KI</string>
   </property>
   <property name="toolTip">
    <string>Coronal View</string>
   </property>
  </action>
  <action name="actionLR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LR</string>
   </property>
   <property name="toolTip">
    <string>Flip left and right</string>
   </property>
  </action>
  <action name="actionUD">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UD</string>
   </property>
   <property name="toolTip">
    <string>Up and Down</string>
   </property>
  </action>
  <action name="actionDualView">
   <property name="text">
    <string>DualView</string>
   </property>
   <property name="toolTip">
    <string>Open dual image viewer</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGraphicsVolumeView</class>
   <extends>QGraphicsView</extends>
   <header>qgraphicsvolumeview.h</header>
  </customwidget>
  <customwidget>
   <class>QDualSlider</class>
   <extends>QSlider</extends>
   <header>qdualslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="bigViewWindow.qrc"/>
 </resources>
 <connections/>
</ui>
